### PR TITLE
auraescript: make helpers.print variadic

### PR DESCRIPTION
**Pull request by jcorbin**
- update examples/cells.ts to demonstrate it
- add editor config so that tsserver stops fighting the project's 4-space indent style

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# top-most EditorConfig file
```
Comment (dmah42) [inline .editorconfig:1]: can we take some from https://github.com/aurae-runtime/ae/blob/main/.editorconfig ?

**Comment by jcorbin** [comment]
> In order to contribute to a Nivenly Foundation project you must [sign and agree to the CLA](https://cla.nivenly.org). Reply with `@cla-bot check` to check again.

@cla-bot check

File: .editorconfig (hunk 1 above)
Comment (jcorbin) [reply .editorconfig:3]: done

**Comment by jcorbin** [comment]
I suppose it'd be nicer to use [Deno.inspect()](https://deno.land/api@v1.29.1?s=Deno.inspect) (which formats like `console.log` would do instead of `JSON.stringify` ; or even to question "hey, why aren't we just using console.log anyhow?"
... but a brief attempt to use either ended up with them being undefined
... will have to look into what's going on with the auraescript deno globals soon, just starting to get my footing here

**Review by dmah42 — APPROVED** [review]

**Comment by dmah42** [comment]
thanks!

**Pull request merged**